### PR TITLE
Release v3.6.2

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Release v3.6.2

#### Changes

The following PRs are considered;
```
* a0612065c	2020-03-19   Merge pull request #2197 from TheThingsNetwork/fix/gs-udp-data
* 966d57aad	2020-03-19   Merge pull request #2186 from rvolosatovs/feature/ns-coverage
* 5edc34e71	2020-03-19   Merge pull request #2188 from rvolosatovs/fix/ns-metrics
* da4c2ac40	2020-03-18   Merge pull request #2161 from TheThingsNetwork/fix/2160-redis-store-error
* 1d4c029cc	2020-03-18   Merge pull request #2183 from TheThingsNetwork/fix/2182-stream-abort
* 144931fe1	2020-03-18   Merge pull request #2168 from TheThingsNetwork/doc/backports
* 1975280f1	2020-03-18   Merge pull request #2176 from rvolosatovs/fix/queue-invalidation
* 6dbc0c4ca	2020-03-18   Merge pull request #2180 from TheThingsNetwork/fix/rpcserver-test-delay
* bcbf601f7	2020-03-18   Merge pull request #2175 from TheThingsNetwork/fix/tx-begin-error
* 8d8634e3e	2020-03-17   Merge pull request #2173 from TheThingsNetwork/fix/2172-events-conn-release
* 2868421fb	2020-03-17   Merge pull request #2174 from rvolosatovs/fix/redis-flaky-test
* 0f25627cb	2020-03-17   Merge pull request #2169 from rvolosatovs/feature/uplink-fixes
* d93b1e62e	2020-03-17   Merge pull request #2164 from TheThingsNetwork/fix/database-uri
* 96ca7072b	2020-03-17   Merge pull request #2165 from TheThingsNetwork/fix/date-time-format
* 1504152f0	2020-03-17   Merge pull request #2158 from TheThingsNetwork/fix/march-2020-vulnerability-fixes
* 3c38f2feb	2020-03-16   Merge pull request #2149 from TheThingsNetwork/issue/2144-suspended-admin
* d6728741b	2020-03-13   Merge pull request #2154 from TheThingsNetwork/fix/2153-bs-ws-panic
* 88bf3f8f0	2020-03-13   Merge pull request #2155 from TheThingsNetwork/quickfix/remove-docker-tags
* 3b549e84c	2020-03-13   Merge pull request #2152 from TheThingsNetwork/fix/devices-cluster-check
* ef3daddf3	2020-03-13   Merge pull request #2145 from rvolosatovs/fix/ns-flow-test
* ad70786c3	2020-03-13   Merge pull request #2142 from rvolosatovs/fix/grpc-gateway-names
```
#### Notes for Reviewers

I forgot to add a ChangeLog line to #2161 so I've added it while bumping the version.

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [ ] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by rvolosatovs** [comment]
Blocked by #2188 #2186

File: pkg/identityserver/store/oauth.go [hunk 1]
```diff
@@ -58,6 +58,8 @@ type AuthorizationCode struct {
 	User   *User
 	UserID string `gorm:"type:UUID;index;not null"`
 
+	UserSessionID *string `gorm:"type:UUID;index"`
```
Comment (johanstokking) [inline pkg/identityserver/store/oauth.go]: This is a `bump/minor` API change

Comment (KrishnaIyer) [reply pkg/identityserver/store/oauth.go]: That's a not an API right? It's an internal model used by the IS. A migration should be sufficient.

Comment (johanstokking) [reply pkg/identityserver/store/oauth.go]: Right, I meant storage. It's not clear in the compatibility commitment, I'll clarify that. But we don't do database migrations in patches.

**Pull request merged**